app/controllers/global_search_controller.rb

```diff
@@ -0,0 +1,14 @@
+class GlobalSearchController < ApplicationController
+
+  customer_tab :all
+  before_filter :authenticate_user!
+  before_filter :check_acting_as
+
+  def index
+    @results = {
```

Comment:
`@results` feels like it's on the verge of graduating from being a `Hash` into some kind of a service object. Maybe when we add another thing to search on…